sec2_the_ruby_programming_language/2.11_oop_intro/1_intro_to_oop.rb

```diff
@@ -0,0 +1,20 @@
+#Introduction to Object Oriented Programming
+class User
+  attr_accessor :name, :email
+  def initialize(name,email)
+    @name = name
+    @email = email
+  end
+
+  def run
+    puts "Hey I'm running"
+  end
+  #The instance of the user is not needed to run this class method
```

Comment:
```suggestion

  #The instance of the user is not needed to run this class method
```
Please care about spacing as well. Having proper spacing allow better readability

Reply:
Note with thankss.